pyp_kcwi/sub_sky.py

```diff
@@ -0,0 +1,142 @@
+# This function is used to subtract sky median
+#
+# INPUT:     - sky_file
+#            - img_file
+#            - mask_limit    mask limit
+#            - cut_ch        cut channel number
+#
+# INPUT EXAMPLE:
+#   median('kb171021_00083_icuber.fits', 'kb171021_00082_icuber.fits', 40, cut_ch=300)
+#
+# MODIFICATION HISTORY:
+#   Written by: Qiong Li
+#   2017-11-01 Initial version
+
+def median(sky_file, img_file, mask_limit, cut_ch=300):
+    from astropy.io import fits
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import matplotlib as mpl
+    sky_input, sh = fits.getdata(sky_file, header=True)
+    img_input, ih = fits.getdata(img_file, header=True)
+
+    img=img_input[cut_ch:(len(img_input[:,1,1])-cut_ch),:,:]
```

Comment:
What are these lines doing?  Add a comment.

Reply:
This is what I suggested to do :-)  We need to cut a few tens layers at the beginning and at the end, because those layers looks very bad. Here we construct a new cube by clipping bad layers at the beginning and the end ("cut_ch" is this meaning, please add this comment to the code)

Reply:
Added.